en/docs/administer/product-security/identity-management-for-the-api-dev-portal.md

```diff
@@ -128,6 +128,12 @@ Follow the instructions below to disable anonymous access to the Developer Porta
 
     [![Disabling Developer Portal Anonymous Mode]({{base_path}}/assets/img/administer/disable-developer-portal-anonymous-mode.png)]({{base_path}}/assets/img/administer/disable-developer-portal-anonymous-mode.png)
 
+    !!! note
+        To disable anonymous access globally, add the following to the `<API-M_HOME>/repository/conf/deployment.toml` file.
+        ```javascript
```

Comment:
Shall we make the type `toml` instead of `javascript`?

Reply:
Done